### DISCUSSION TITLE
What exactly does "Done" do?

### DISCUSSION THREAD

**Discussion by judofyr** [colinlienard/gitlight#281] [Q&A]
I assume that marking an issue as "Done" makes sure that it won't be marked as "Unread" on every activity, but are there some cases where it will actually go back to "Unread"? E.g. what if someone mentions me?

**Comment by colinlienard** [comment]
Marking as done does exactly the same thing as marking as read, in addition to removing the issue from the interface. So If an activity occurs, the issue will come back as unread.
If you don't want an issue to come back every time, you can mute it like this:

![image](https://github.com/colinlienard/gitlight/assets/64312634/8b860e19-5bfc-46a2-98c1-5407cb7a27ab)

You can also mute persons (bots for example) or entire repositories if you want
